coral/src/app/layout/main-navigation/MainNavigation.tsx

```diff
@@ -46,8 +46,12 @@ function MainNavigation() {
               }
             />
             <MainNavigationLink
-              to={`/myTopicRequests`}
               linkText={"My team's requests"}
+              to={Routes.REQUESTS}
+              active={
+                pathname.startsWith(Routes.REQUESTS) ||
+                pathname.startsWith("/request")
```

Comment:
I think `pathname.startsWith("/request")` is not necessary. It would imply that we have individual routes like `request/12/overview` or something, and I don't think we have any plan for this :o

Reply:
Mainly copy+pasta from the approvals. Was not sure so "left" it there. Can remove it while adding the tests :)